src/app/Models/constants.ts

```diff
@@ -2,6 +2,7 @@ export const LOGIN = 'login';
 export const REGISTER = 'register';
 export const HOME = 'home';
 export const USER = 'me';
+export const BLOGID = 'blog/:id';
```

Comment:
we can use functions to get the link with a id.
const getBlog = (id) => `blog/${id}`;

Reply:
Bhai this is just `queryparam` declaration like a place holder and need this const only for declaring routes. I am created the function like you suggested for the `routerLink` for linking in the component.html files. Thanks bhai.